### ISSUE TITLE
"date_paid" with YYYY-MM-DD

### ISSUE THREAD

**Issue by bschweighofer** [opencost-de/opencost#9]
Currently, "date_paid" is mandatory with the specification YYYY-MM-DD. However, this is not always feasible, often only the year is known (more detailed information would have to be requested from the administration by individual inquiry). In addition, at the year transition it is sometimes not easy to see if it was paid from the remaining funds of the previous year. Would it be possible to make it "non-required" here?

ReportBy: email

**Comment by aw-bib** [comment]
This was addressed in PR #3 where we relaxed the date requirement to be `YYYY` (only a year), `YYYY-MM` (a year and a month) or `YYYY-MM-DD` the full date.

From the discussion it became clear that no date at all might not do.

However, we also see that especially for transformative agreements or flatrates the `date_paid` becomes an issue. Especially, publications early in the year can sometimes not easily associated to a `date_paid`, e.g. if the publication was accepted in October and only published in March of the subsequent year.

This might need further discussion.
